django/thunderstore/repository/views/team_settings.py

```diff
@@ -276,5 +276,8 @@ def get_success_url(self):
         return self.object.donation_link_url
 
     def form_valid(self, form: DonationLinkTeamForm):
+        self.object = form.save()
+        if form.errors:
+            return super().form_invalid(form)
```

Comment:
Doesn't the form view by default call `form.save()` already in the super call, so is this change required at all? I suppose it can bubble up the errors better here, but I'd imagine it's also double-calling the save now if it's left as is.